src/char/char.h

```diff
@@ -29,6 +29,19 @@ enum {
 	TABLE_GUILD_STORAGE,
 };
 
+typedef enum e_makechar_error {
+	ERROR_MK_SQL = -255,
```

Comment:
Yes but I don't like make, as it's more make as DO stuff, so I would name it MAKECHAR_DO_SQL